examples/hello_world/android/settings.gradle

```diff
@@ -6,16 +6,29 @@
 // To update all the settings.gradle files in the Flutter repo,
 // See dev/tools/bin/generate_gradle_lockfiles.dart.
 
+pluginManagement {
+    def localPropertiesFile = new File(rootProject.projectDir, "local.properties")
+    def properties = new Properties()
+    assert localPropertiesFile.exists()
+    localPropertiesFile.withReader("UTF-8") { reader -> properties.load(reader) }
+    def flutterSdkPath = properties.getProperty("flutter.sdk")
```

Comment:
Does not need to happen in this pr but I think we should make the variables we store in local.properties constants that are defined on the flutter plugin like `flutter.minSdkVersion`. 
I know the values should not change but I like that constants help point to the code that uses/defines those values.

Reply:
+1 to this.

Unfortunately, FGP (Flutter Gradle Plugin) is not yet available when `settings.gradle` is being evaluated.

Another plugin (a settings plugin, which is available when settings are evaluated) would have to be created, just to hold these constants.

Other ways to do this probably exist, but as of now I'm not aware of any.